mergify_engine/tasks/engine/v2.py

```diff
@@ -111,6 +111,15 @@ def gen_summary(event_type, data, pull, match):
     summary += gen_summary_rules(match.matching_rules)
     ignored_rules = len(list(filter(lambda x: not x[0]['hidden'],
                                     match.ignored_rules)))
+
+    commit_message = pull.get_merge_commit_message()
```

Comment:
Should that check for the method too?

We could set a warning if somebody uses `rebase` and set a commit message.

Reply:
At this point we have no idea which method will be used as the configuration can have multiple rules that may merge the PR with different method.